### DISCUSSION TITLE
2021-08-11 Weekly Meeting 16 Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#74] [Weekly Meetings]
Weekly Meeting 16 is happening on Wednesday 2021-08-11 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- Welcome new members, review process for Submission and forming working groups 
- Website
- glTF Sub Group
  - OMI Audio Emitter Extension
  - Blender Tooling
  - Hubs -> OMI glTF Transform CLI
- Discord Bot
- World Exploration Meetup Recap
- Demo Days
- Stellar Mesh

Add ons: 
- Evo suggested we talk about the nvidia announcement regarding [omniverse](https://80.lv/articles/nvidia-announces-a-major-expansion-of-nvidia-omniverse/)

**Comment by MauiJerry** [comment]
Key part of nVidia Omniverse is the use of Pixar Universal Scene Description (USD) as the exchange format. They announced this week that Blender will support it.  They have bunch of announcements this week, including live talk while this meeting is happening (i think)

**Reply by mrmetaverse** [reply]
Exciting times indeed ☺️. It is amazing what unlocks when groups collaborate and focus on the humans involved. 

https://venturebeat.com/2021/08/10/nvidia-opens-its-metaverse-for-engineers-by-adding-millions-of-blender-users-to-omniverse/

**Comment by technobaboo** [comment]
Don't forget showing off OMI-chan to everyone and asking for feedback, should take ~5 minutes

**Comment by mrmetaverse** [comment]
### Working agreements 
- if your working group would like to create a team working agreement, please reach out to Jesse (@mrmetaverse) for facilitation (if needed)
- We have found that creating a working agreement for all of OMI may not be ideal, as working agreements are best used for teams working on a specific topic. So for the gltf working group, or the upcoming stellar mesh working group, I recommend we draft working agreements that fit each teams. For example, we may have a working group consist mainly of people in Europe and they may want to establish standard working hours, or a standard toolset for workshops, etc. 
- Working agreements help strengthen team dynamics, and provide a psychologically safe place for all to participate. 
- Working agreements are also ideal for establish ceremony cadence, i.e. if your team wants to follow Scrum, you may want to establish a daily standup meeting time, a demo day, and a backlog refinement. But if your team doesn't want to use scrum, you may want to establish kanban or some other method of sharing and tracking work. 
- The most important part is that we should know how work gets done, how to get involved, and how to respect each other. 
-

**Comment by mrmetaverse** [comment]
Notes for todays session are started here on [hackmd](https://hackmd.io/@mrmetaverse/omi-081121)
